### ISSUE TITLE
Killing and re-launching eth appears to clear pending transactions

### ISSUE THREAD

**Issue by halfalicious** [ethereum/cpp-ethereum#5019]
If one sends out a transaction and then kills and re-launches eth, `web3.eth.pendingTransactions` will return an empty array. Geth will not. 

Note that I've verified that `web3.eth.pendingTransactions` returns the newly submitted transaction before I kill and re-launch eth so this isn't a case of `web3.eth.pendingTransactions` simply being broken (note that I'm using a locally-built version of eth which includes the changes from PR #5018).

**Comment by gumb0** [comment]
Yeah we don't save pending transaction between the runs of the client. Geth probably does.
After re-launch we can get pending transactions only from other nodes.
